getting-started/templates/systemlink-secrets.yaml

```diff
@@ -389,3 +389,21 @@ routineservice:
       ## Refer to the MongoDB documentation for key generation: https://www.mongodb.com/docs/manual/tutorial/enforce-keyfile-access-control-in-existing-replica-set/#create-a-keyfile
       ##
       replicaSetKey: ""  # <ATTENTION>
+
+## Secret configuration for licensing
```

Comment:
We decided to wait till the beginning of next month, to complete this PR.

Since we do not have much time left in this iteration, we will focus on adding the License Service to the umbrella chart, during the beginning of next month.